dependency/vault_read_test.go

```diff
@@ -83,11 +83,20 @@ func TestNewVaultReadQuery(t *testing.T) {
 	}
 }
 
-func TestVaultReadQuery_Fetch(t *testing.T) {
+func TestVaultReadQuery_Fetch_KVv1(t *testing.T) {
 	t.Parallel()
 
-	clients, vault := testVaultServer(t)
-	defer vault.Stop()
+	clients, vault := testVaultServer("read_fetch_v1", "1")
+	secrets_path := vault.secrets_path
```

Comment:
snake case -> camel case

Reply:
Changing the struct variable to camelCase... leaving local alone.